### PR TITLE
Feature summary

### PR DESCRIPTION
**Pull request by hvyavuno**
Add function for new endpoint:

`{{api_host}}/sample/v1/projects/invoices/summary?startDate=2020-01-01&endDate=2020-02-01&extProjectId=**** `

### CODE REVIEW

File: dynatademand/schemas/request/query/get_invoices_summary.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (dynata-bradley) [inline dynatademand/schemas/request/query/get_invoices_summary.json]: Is there no schema for this in the docs yet?

**Comment by dynata-bradley** [comment]
It looks like there are linting failures. Our automated tests are using [this](https://github.com/dynata/python-demandapi-client/blob/e8a606e22111155065f28b26f7e7bc7970af8320/.github/workflows/pythonpackage.yml#L30) flake8 statement. Tests pass though.

File: tests/test_invoices_summary.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
```
Comment (dynata-bradley) [inline tests/test_invoices_summary.py]: Why not add this test to `test_invoices.py`?

File: dynatademand/schemas/request/query/get_invoices_summary.json (hunk 1 above)
Comment (hvyavuno) [reply dynatademand/schemas/request/query/get_invoices_summary.json]: I don't know what schema in this context means. Perhaps `startDate`, `endDate` and `extProjectId`?

File: tests/test_line_items.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# encoding: utf-8
 from __future__ import unicode_literals, print_function
+# encoding: utf-8
```
Comment (dynata-bradley) [inline tests/test_line_items.py]: This change doesn't appear related to this PR.

File: tests/test_invoices_summary.py (hunk 2 above)
Comment (hvyavuno) [reply tests/test_invoices_summary.py]: Oh yes, makes a lot of sense. Thank you!

File: tests/test_line_items.py (hunk 3 above)
Comment (hvyavuno) [reply tests/test_line_items.py]: Yes. I ran the tests on my local machine and it said to move `__future__` to the top first. What would you like me to do?

File: dynatademand/schemas/request/query/get_invoices_summary.json (hunk 1 above)
Comment (dynata-bradley) [reply dynatademand/schemas/request/query/get_invoices_summary.json]: From the docs. For example, we got the query schema for get attributes from 

https://developers.dynata.com/demand-api-reference/data_endpoints/attributes/get-attributes

by clicking on the copy button at the right of "query parameters".

Comment (dynata-bradley) [reply dynatademand/schemas/request/query/get_invoices_summary.json]: The black thing atop the text here: 
Screen Shot 2020-01-23 at 3 35 49 PM

**Comment by dynata-bradley** [comment]
We also want to add a link for this function to the docs in the README, but I don't see this endpoint in the docs yet?

File: dynatademand/schemas/request/query/get_invoices_summary.json (hunk 1 above)
Comment (hvyavuno) [reply dynatademand/schemas/request/query/get_invoices_summary.json]: Okay. Will read now.

File: tests/test_line_items.py (hunk 3 above)
Comment (dynata-bradley) [reply tests/test_line_items.py]: All the other tests start with the encoding line, so I have no idea why this one would have been singled out. Can you post the full output from `pytest` where you saw that?

**Comment by DynataRidley** [comment]
I'm personally OK with including the function without a docs link as long as we have a usage example included in the README. Other than that, I think the only things holding this back are the linting issues and the comments Bradley mentioned above.

File: tests/test_line_items.py (hunk 3 above)
Comment (hvyavuno) [reply tests/test_line_items.py]: I am sorry. I can't reproduce it.
